packit/utils/specfile_helper.py

```diff
@@ -0,0 +1,115 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import logging
+from typing import List, Optional
+
+from specfile import Specfile
+from specfile.exceptions import SpecfileException
+
+from packit.patches import PatchMetadata
+from packit.exceptions import PackitException
+
+try:
+    from rebasehelper.plugins.plugin_manager import plugin_manager
+except ImportError:
+    from rebasehelper.versioneer import versioneers_runner
+
+
+logger = logging.getLogger(__name__)
+
+
+class SpecfileHelper:
```

Comment:
Do you think we can possibly move this functionality to the spec-file library or is it too packit-specific?

Reply:
I think it could be possible, except for `get_upstream_version()`, such functionality doesn't belong to specfile library IMHO.

Reply:
Yes, `get_upstream_version` is really Packit/Anytia/RebaseHelper specific...